### DISCUSSION TITLE
Is documentation on "Effects" misleading?

### DISCUSSION THREAD

**Discussion by rlaffers** [statelyai/xstate#1286] [General] [answered]
The [xstate docs](https://xstate.js.org/docs/guides/effects.html) say this:

> "Fire-and-forget" effects, which execute a side-effect and **do not send any events** back to the statechart:
> * Actions
> * Activities

However, some built-in action creators do in fact send events: send, forwardTo, respond, escalate.
Should the documentation be updated, or am I misinterpreting what it says?

**Comment by davidkpiano** [comment, marked as answer]
It should be updated, yes. I would appreciate a PR if you'd like!

**Reply by rlaffers** [reply]
PR #1288 
I tried to be more specific about what actions are used for. Hope it makes sense.
